projects/ffmpeg/build.sh

```diff
@@ -193,6 +193,12 @@ rm `find fate-suite -name '*.pcm'`
 
 zip -r $OUT/${fuzzer_name}_seed_corpus.zip fate-suite
 
+# Build fuzzer for demuxer fed at IO level
+fuzzer_name=ffmpeg_IO_DEMUXER_fuzzer
+echo -en "[libfuzzer]\nmax_len = 1000000\n" > $OUT/${fuzzer_name}.options
```

Comment:
Note that this size restriction won't work for afl and honggfuzz fuzzers, it is advised to check size in LLVMTestOneInput of the fuzzer
if (size > 1000000)
  return

Reply:
hmm the size restriction is just there because its there in all the others in ffmpeg too. 
Looking at the history it was added in 03417736d0aee695fee9689cc845b8d51cf10eb4.
If it is still needed i can add it in LLVMFuzzerTestOneInput, otherwise it can be removed

Reply:
Yes we don't recommend that anymore, so all fuzzers should just have it in LLVMFuzzerTestOneInput to work across all fuzzing engines properly.